Content.Client/Access/UI/IdCardConsoleBoundUserInterface.cs

```diff
@@ -1,6 +1,5 @@
 using Content.Shared.Access;
 using Content.Shared.Access.Components;
-using Content.Shared.Access;
```

Comment:
was this intended?

Reply:
heh, I do not know, it's just that visual studio was constantly swearing at it, and I decided to delete :D